platform/ios/src/MGLAnnotationView_Private.h

```diff
@@ -0,0 +1,23 @@
+#import "MGLAnnotationView.h"
+#import "MGLAnnotation.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLAnnotationViewDelegate <NSObject>
+
+- (void)annotationView:(MGLAnnotationView *)annotationView didReceiveTapAtPoint:(CGPoint)point;
+
+@end
+
+@interface MGLAnnotationView (Private)
+
+@property (nonatomic, weak) id<MGLAnnotationViewDelegate> delegate;
+
+// Identifier used to specifiy a reuse queue for the annotation view type.
+@property (nonatomic, copy, nullable) NSString *reuseIdentifier;
+
+@property (nonatomic) id<MGLAnnotation> annotation;
```

Comment:
MKAnnotationView exposes this property publicly. Is there a reason for keeping it private?